bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -409,6 +409,14 @@ public void handle(PullRequestBot bot, HostedCommit commit, LimitedCensusInstanc
                           "$ git commit --message 'Describe additional changes made'\n" +
                           "$ git push " + fork.url() + " " + backportBranchName + "\n" +
                           "```");
+            if (bot.repo().forge().name().equals("GitHub")) {
+                reply.println();
+                reply.print("@");
+                reply.print(realUser.username());
+                reply.print(" ");
```

Comment:
If the backport command was used in a pull request, in the beginning of the reply message, the bot will @`the author of the pull request` and @`the command issuer`. 
To be clear, I would like to @`the command issuer` again before the reminder.